cmd/explaintest/r/index_merge.result

```diff
@@ -0,0 +1,116 @@
+drop table if exists t1;
```

Comment:
Seems that we would add the same tests if we want to add some tests in the executor package.
So we don't add that this time?
Or add these to the executor package so the tests can be run parellelly.

Reply:
Of course.